spec/android_audio_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'device_api/android'
+require_relative 'helpers/stub_helper_spec'
+
+include RSpec
+
+describe DeviceAPI::Android::Plugin::Audio do
+  describe 'Audio functions' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.